packages/thicket-webapp/README.md

```diff
@@ -10,7 +10,13 @@ By setting the corresponding `env` var the IPFS instance will connect to the def
 - `WRTC_IPFS` - will use IPFS’s [`webrtc-star-signal`](https://wrtc-star.discovery.libp2p.io) server. Default value if no value is set.
 
 ```sh
+# to be sure have nodejs 8
```

Comment:
Would you please double check this?
I ran `nvm use 9` then `yarn && yarn bootstrap` and it worked correctly.
(it used node v9.11.2)

Reply:
<img width="807" alt="thicket-docker" src="https://user-images.githubusercontent.com/3072458/42682554-33de9ba6-868b-11e8-8f14-0233099723df.png">

I double checked with a setting similar to yours (using docker image for node9)